src/AddressRegistry/StreetName/Commands/ProposeAddress.cs

```diff
@@ -12,6 +12,7 @@ public class ProposeAddress : IHasCommandProvenance
 
         public StreetNamePersistentLocalId StreetNamePersistentLocalId { get; set; }
         public PostalCode PostalCode { get; set; }
+        public MunicipalityId PostalCodeMunicipalityId { get; set; }
```

Comment:
Dit is een verwarrende naamgeving voor mij, het is een combinatie van PostalCode en MunicipalityId?

Reply:
`PostalInfoMunicipalityId` might be better?